mfr/providers/osf/provider.py

```diff
@@ -13,7 +13,10 @@
 
 from mfr.core import exceptions
 from mfr.core import provider
+from mfr.core.utils import sizeof_fmt
 from mfr.providers.osf import settings
+from mfr.settings import MAX_FILE_SIZE_TO_RENDER
+from mfr.extensions.tabular.exceptions import TooBigToRenderError
```

Comment:
Validating in the provider is a good idea, but in that case this error should be a TabualrError.  Make it a subclass of ProviderError instead.